lib/board.rb

```diff
@@ -20,6 +20,7 @@ def empty_cells
 
   def update_board(position, round)
     @board[position] = (round.even? ? 'X' : 'O')
+    @board[position]
```

Comment:
- [ ] **OPTIONAL** It is not necessary to return @board[position] again since ruby returns the last line by default, you may delete this line